vir/src/debug_info.rs

```diff
@@ -0,0 +1,135 @@
+#[cfg(feature="vir_debug")]
```

Comment:
Would it make sense to organise these pair-wise (imports for `vir_debug`, without; definition of `DebugInfo` for `vir_debug`, without; etc) using `cfg_if`?

Reply:
Ah, yes, that would be better. I'll update it accordingly.